From: Allen Winter <allen.winter@kdab.com>
Date: Sat, 09 Jun 2012 17:03:12 +0000
Subject: remove REQUIRED from the find_package(KActivities).
X-Git-Url: http://quickgit.kde.org/?p=kde-runtime.git&amp;a=commitdiff&amp;h=a7766cdcd1be164d1cda35df959c1fea1136decc
---
remove REQUIRED from the find_package(KActivities).
then tell the macro_log_feature that KActivities is required.
this way, if people are missing KActivities they will get a nice log
message about it.
---


--- a/plasma/declarativeimports/plasmaextracomponents/CMakeLists.txt
+++ b/plasma/declarativeimports/plasmaextracomponents/CMakeLists.txt
@@ -1,7 +1,7 @@
 project(plasmaextracomponents)
 
-find_package(KActivities REQUIRED NO_MODULE)
-macro_log_feature(KACTIVITIES_FOUND "KActivities" "Library and infrasturcture for using Activities from applications" "http://kde.org/" FALSE "" "Provides the ability to access activities from QML.")
+find_package(KActivities)
+macro_log_feature(KACTIVITIES_FOUND "KActivities" "Library and infrasturcture for using Activities from applications" "git://anongit.kde.org/kactivities" TRUE "" "Provides the ability to access activities from QML.")
 
 #include(KDE4Defaults)
 

